schema/answers/checkbox.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "answer": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/id"
+      },
+      "q_code": {
+        "$ref": "../common_definitions.json#/q_code"
+      },
+      "label": {
+        "type": "string"
+      },
+      "guidance": {
+        "$ref": "../common_definitions.json#/guidance"
+      },
+      "description": {
+        "type": "string"
+      },
+      "type": {
+        "type": "string",
+        "pattern": "^Checkbox$"
```

Comment:
Did you consider using an enum of one value for types rather than a regex? I think `"enum": ["Checkbox"]` reads better as it's on a single line and there's no regex characters.